go/vt/tabletserver/splitquery/equal_splits_algorithm.go

```diff
@@ -38,7 +38,10 @@ type EqualSplitsAlgorithm struct {
 func NewEqualSplitsAlgorithm(splitParams *SplitParams, sqlExecuter SQLExecuter) (
 	*EqualSplitsAlgorithm, error) {
 
-	assertEqual(len(splitParams.splitColumns), len(splitParams.splitColumnTypes))
+	if len(splitParams.splitColumns) != len(splitParams.splitColumnTypes) {
+		panic(fmt.Sprintf("len(splitparams.splitColumns) != len(splitparams.splitColumnTypes): %v!=%v",
```

Comment:
I thought we agreed bad user input should return an error, not panic, if possible?